### PR TITLE
Feature user profile update

### PR DESCRIPTION
**Pull request by willkoua**
| Q                                          | R
| ------------------------------------------ | -------------------------------------------
| Type of contribution ?                     | [Features]
| Tickets (_issues_) concerned               | [TV-165]

---

##### What have you changed ?
update edit profile module

##### How did you change it ?
the user can updtae email and university

##### Is there a special consideration?
...

Verification :
 -  [X] My branch name respect the standard defined in `CONTRIBUTING.md`
 -  [X] All my commits respect the standard defined in `CONTRIBUTING.md`
 -  [X] My additions are I18N
 -  [X] This Pull-Request fully meets the requirements defined in the issue
 -  [X] I added or modified the attached tests
 -  [ ] I added or modified the documentation

[TV-165]: https://fjnr-inc.atlassian.net/browse/TV-165

### CODE REVIEW

File: retirement/tests/tests_viewset_Reservation.py [hunk 1]
```diff
@@ -52,6 +52,7 @@ class ReservationTests(APITestCase):
     @classmethod
     def setUpClass(cls):
         super(ReservationTests, cls).setUpClass()
+        cls.maxDiff=None
```
Comment (RignonNoel) [inline retirement/tests/tests_viewset_Reservation.py]: Please remove this since it's just for your local test

File: store/serializers.py [hunk 2]
```diff
@@ -461,7 +461,11 @@ def create(self, validated_data):
                     ]
                     coupon_info['orderline'].save()
                 else:
-                    raise serializers.ValidationError(coupon_info['error'])
+                    raise serializers.ValidationError({
+                        "non_field_errors": [
+                            coupon_info['error']
+                        ],
+                    })
```
Comment (RignonNoel) [inline store/serializers.py]: Not related to this PR, please remove this

**Review by RignonNoel — CHANGES_REQUESTED** [review]

File: retirement/tests/tests_viewset_Reservation.py (hunk 1 above)
Comment (willkoua) [reply retirement/tests/tests_viewset_Reservation.py]: done

File: store/serializers.py (hunk 2 above)
Comment (willkoua) [reply store/serializers.py]: done

**Comment by RignonNoel** [comment]
`.env.example` is not updated with your new settings `RESERVATION_CANCELLED`. Please update the example to help maintainers do their jobs.

**Review by RignonNoel — APPROVED** [review]

**Pull request merged**